optaweb-tsp-planner/pom.xml

```diff
@@ -54,12 +55,43 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>include-ui</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>org.optaweb.tsp</groupId>
+          <artifactId>optaweb-tsp-ui</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+  </profiles>
```

Comment:
@danielefiungo What is the reason for having the dependency in a profile?

Reply:
The day UI can be deployed on a simple http server and include static resources inside BE can be optionally disabled in CI when UI is deployed separately.

@yurloc Make sense for you?